auction-server/src/auction/service/auction_manager.rs

```diff
@@ -370,20 +410,26 @@ impl AuctionManager<Svm> for Service<Svm> {
                         transaction: bid.chain_data.transaction.clone(),
                     },
                 })
-                .await
-            {
+                .await;
+            results.push(result.clone());
+            match result {
                 Err(RestError::SimulationError {
                     result: _,
                     reason: _,
                 }) => {}
                 // Either simulation was successful or we can't simulate at this moment
-                _ => return Ok(vec![bid.clone()]),
+                _ => {
+                    tracing::Span::current().record("simulation_result", format!("{:?}", result));
+                    return Ok(vec![bid.clone()]);
+                }
             }
         }
+
+        tracing::Span::current().record("simulation_result", format!("{:?}", results));
```

Comment:
it seems that the default span attribute value length limit is around 2048 bytes. Maybe we should use a more compact representation here?